Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-settings.jsp

```diff
@@ -12,19 +12,23 @@
 <div>
     <ul>
         <li><a
-            href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${project.projectId}"><span
+            href="${pageContext.servletContext.contextPath}/auth/workbench/projects/${publicpageprojectid}"><span
```

Comment:
After updating statistics settings, I need to add project object in my controller, which is an additional call as I am not doing anything with project object except projectid